Sources/SwiftGenKit/Parsers/Strings/FileTypeParser/StringsDictFileParser.swift

```diff
@@ -0,0 +1,58 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+extension Strings {
+  final class StringsDictFileParser: StringsFileTypeParser {
+    private let options: ParserOptionValues
+
+    let shouldOverwriteValuesInExistingTable: Bool = true
+
+    init(options: ParserOptionValues) {
+      self.options = options
+    }
+
+    static let extensions = ["stringsdict"]
+    static let allOptions: ParserOptionList = [Option.separator]
+    static let propertyListDecoder = PropertyListDecoder()
+
+    func parseFile(at path: Path) throws -> [Strings.Entry] {
+      guard let data = try? path.read() else {
+        throw ParserError.failureOnLoading(path: path.string)
+      }
+
+      do {
+        let plurals = try Strings.StringsDictFileParser.propertyListDecoder
+          .decode([String: StringsDict].self, from: data)
+          .compactMapValues { stringsDict -> StringsDict.PluralEntry? in
+            guard case let .pluralEntry(pluralEntry) = stringsDict else { return nil }
```

Comment:
nit: add code comment
```suggestion
            // We only support .pluralEntry (and not .variableWidthEntry) for now, so filter out the rest
            guard case let .pluralEntry(pluralEntry) = stringsDict else { return nil }
```